HotTips/Tips/EDI_FullScreen.md

```diff
@@ -0,0 +1 @@
+﻿Press Ctrl + Backspace to delete the word to left of the cursor.
```

Comment:
I think this is meant to say "Press **Shift+Alt*Enter** to enter "Full Screen" mode.